src/ios/SimpleCameraPreview.m

```diff
@@ -358,4 +358,104 @@ - (void)deallocateMemory {
     locationManager = nil;
 }
 
+- (void) initVideoCallback:(CDVInvokedUrlCommand*)command {
+    self.videoCallbackContext = command;
+    NSDictionary *data = @{ @"videoCallbackInitialized" : @true };
+    
+    CDVPluginResult *pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsDictionary:data];
+    [pluginResult setKeepCallbackAsBool:true];
+    [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
+}
+
+- (void)startVideoCapture:(CDVInvokedUrlCommand*)command {
+    if (self.sessionManager != nil && !self.sessionManager.movieFileOutput.isRecording) {
```

Comment:
I think we can invert the condition and make an early return instead of having an `if` and an `else` block
```objective-c
if (self.sessionManager == nil && self.sessionManager.movieFileOutput.isRecording) {
  CDVPluginResult *pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsString:@"Session not initialized or already recording"];
  [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
  return;
}
// other logic
```

Reply:
Should the condition be an `||` condition?